Bio/SearchIO/HmmerIO/hmmer3_tab.py

```diff
@@ -42,11 +42,12 @@ def _parse_row(self):
         # combine them all into one string in the 19th column
         if len(cols) > 19:
             cols[18] = ' '.join(cols[18:])
-        # if it's < 19, we have no description columns, so use an empty string
+        # if it's 18, we have no description columns, so use an empty string
         # instead
-        elif len(cols) < 19:
+        elif len(cols) == 18:
             cols.append('')
-            assert len(cols) == 19
+        else:
```

Comment:
This is slightly different than the previous check, which does nothing when `len(cols) == 19`. Perhaps this can be `elif len(cols) < 18` instead of `else`. The tests seem to detect this as well.

On a related note, it seems ok if the entire loop is re-written also. Something like:

```
cols = [x for x in self.line.strip().split(' ') if x]
if len(cols) < 18:
    raise ValueError("Less columns than expected, only %i" % len(cols)
cols[18] = '  '.join(cols[18:])    # which has the same effect as the two, previous if-clauses.
```

Reply:
You're right, I missed the case when there already where 19 columns.

Reply:
Applied your alternative which is much shorter :)

Reply:
Thank you :). I made a slight mistake in the `.join` call, I now realize. It should be `' '` (single space), instead of `'  '` (double space). That's why the tests is still failing now. After this is fixed, it should be good to merge ~

Reply:
Thanks - I didn't see that when I copy-and-pasted it.